test/sanity/ignore.txt

```diff
@@ -2111,13 +2111,29 @@ lib/ansible/modules/cloud/vmware/vsphere_copy.py validate-modules:undocumented-p
 lib/ansible/modules/cloud/vmware/vsphere_copy.py validate-modules:doc-missing-type
 lib/ansible/modules/cloud/vultr/vultr_block_storage.py validate-modules:parameter-type-not-in-doc
 lib/ansible/modules/cloud/vultr/vultr_block_storage.py validate-modules:doc-missing-type
+lib/ansible/modules/cloud/vultr/_vultr_block_storage_facts.py validate-modules:return-syntax-error
```

Comment:
I thought the fixes in this PR would bring these modules into compliance. Is that not the case?

Reply:
Not completely: `type: complex` requires `contains:`, but the `contains:` provided should have been `sample:`/`example:`, so fixing the latter adds a new problem which has been hidden before.

Reply:
Ah, that makes sense. Thanks.